users/users-model.js

```diff
@@ -0,0 +1,35 @@
+const db = require('../database/dbConfig.js');
+
+module.exports = {
+  add,
+  find,
+  findBy,
+  findById,
+  findByDepartment
+};
+
+function find() {
+  return db('users').select('id', 'username', 'password');
```

Comment:
Do you mean to limit your query to not include department here?
You could get all columns with:
```javascript
  return db('users');
```
since it'll return all columns by default if you don't specify any with a `select`.